edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/ObjectToStringConverter.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes
+{
+    using System;
+    using System.IO;
+    using Newtonsoft.Json;
+
+    public class ObjectToStringConverter<T> : JsonConverter<T>
+    {
+        public override void WriteJson(JsonWriter writer, T value, JsonSerializer serializer)
+        {
+            using (var text = new StringWriter())
+            {
+                serializer.Serialize(text, value);
+                writer.WriteValue(text.ToString());
+            }
+        }
+
+        public override T ReadJson(JsonReader reader, Type objectType, T existingValue, bool hasExistingValue, JsonSerializer serializer)
+        {
+            using (var text = new StringReader(reader.Value.ToString()))
```

Comment:
>Value [](start = 54, length = 5)

Can value be null?

Reply:
No, null value filtered out on upper level.